zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/ZbColumnFamilies.java

```diff
@@ -171,80 +173,92 @@ public enum ZbColumnFamilies implements EnumValue {
   USER_TASK_STATES(80),
   COMPENSATION_SUBSCRIPTION(81),
 
-  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_DEPLOYMENT_KEY(82),
-  DMN_DECISION_KEY_BY_DECISION_ID_AND_DEPLOYMENT_KEY(83),
-  FORM_KEY_BY_FORM_ID_AND_DEPLOYMENT_KEY(84),
+  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_DEPLOYMENT_KEY(82, ColumnFamilyScope.GLOBAL),
+  DMN_DECISION_KEY_BY_DECISION_ID_AND_DEPLOYMENT_KEY(83, ColumnFamilyScope.GLOBAL),
+  FORM_KEY_BY_FORM_ID_AND_DEPLOYMENT_KEY(84, ColumnFamilyScope.GLOBAL),
 
   MESSAGE_CORRELATION(85),
 
-  USERS(86),
+  USERS(86, ColumnFamilyScope.GLOBAL),
   @Deprecated
-  USER_KEY_BY_USERNAME(87),
+  USER_KEY_BY_USERNAME(87, ColumnFamilyScope.GLOBAL),
 
-  CLOCK(88),
+  CLOCK(88, ColumnFamilyScope.GLOBAL),
 
-  AUTHORIZATIONS(89),
-  PERMISSIONS(90),
+  AUTHORIZATIONS(89, ColumnFamilyScope.GLOBAL),
+  PERMISSIONS(90, ColumnFamilyScope.GLOBAL),
 
-  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_VERSION_TAG(91),
-  DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION_TAG(92),
-  FORM_KEY_BY_FORM_ID_AND_VERSION_TAG(93),
+  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_VERSION_TAG(91, ColumnFamilyScope.GLOBAL),
+  DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION_TAG(92, ColumnFamilyScope.GLOBAL),
+  FORM_KEY_BY_FORM_ID_AND_VERSION_TAG(93, ColumnFamilyScope.GLOBAL),
 
   @Deprecated
-  AUTHORIZATION_KEY_BY_RESOURCE_ID(94),
+  AUTHORIZATION_KEY_BY_RESOURCE_ID(94, ColumnFamilyScope.GLOBAL),
   @Deprecated
-  OWNER_TYPE_BY_OWNER_KEY(95),
+  OWNER_TYPE_BY_OWNER_KEY(95, ColumnFamilyScope.GLOBAL),
 
-  ROUTING(96),
+  ROUTING(96, ColumnFamilyScope.GLOBAL),
```

Comment:
❓ Should the be on the new partition? Pingin @lenaschoenburg for this one. I can see the `ScaleUpProcessor` doesn't distribute anything so it'll only be on 1 partition. Therefore, I believe this shouldn't be global.

Reply:
I think it should be distributed over all partitions, it may have not yet implemented.

Reply:
@lenaschoenburg good point, it can be initialized via distributed commands once the partition is initialized